config/wrapper/pci_input.conf

```diff
@@ -1,20 +1,57 @@
 [io_nvme_fvt]
+devices = interfaces:0
```

Comment:
interfaces for nvme bucket, any test needs an interace?

Reply:
Hi Abdul, Thanks for the review.  This is based on the existing input file which was looking for values such as nvme0, nvme1.   which is provided by the interfaces parameter.

Reply:
https://github.com/lop-devops/tests/blob/47de43e0efad17c1eb23685dd864d42e84d0d288/config/inputs/io_nvme_input.txt#L2

the nvme value is taken as device: and not as interfaces   in input file  ..can you check once again

Reply:
From input file perspective, the terms,
a. device means block device
b. interface means network interface

inside the code,  the term interface is generic,  it captures the data of both block and network device.  
It has been coded in such a way that,  depending on the device type the corresponding value is gathered.